Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -930,70 +730,126 @@ void ExecuteCommand(u32 command_0, u32 command_1, u32 command_2, u32 output_addr
+    DolphinAnalytics::Instance().ReportGameQuirk(GameQuirk::USES_DVD_LOW_READ_DISK_BCA);
+    // NSMBW checks that the first 0x33 bytes of the BCA are 0, then it expects a 1.
+    // Most (all?) other games have 0x34 0's at the start of the BCA, but don't actually
+    // read it.  NSMBW doesn't care about the other 12 bytes (which contain manufacturing data?)
+
+    // TODO: Read the .bca file that cleanrip generates, if it exists
+    // Memory::CopyToEmu(output_address, bca_data, 0x40);
+    Memory::Memset(output_address, 0, 0x40);
+    Memory::Write_U8(1, output_address + 0x33);
     break;
   // Probably only used by Wii
-  case DVDLowRequestDiscStatus:
+  case DICommand::REQUEST_DISC_STATUS:
     ERROR_LOG(DVDINTERFACE, "DVDLowRequestDiscStatus");
+    DolphinAnalytics::Instance().ReportGameQuirk(GameQuirk::USES_DVD_LOW_REQUEST_DISC_STATUS);
+    SetHighError(ERROR_INV_CMD);
+    interrupt_type = INT_DEINT;
     break;
   // Probably only used by Wii
-  case DVDLowRequestRetryNumber:
+  case DICommand::REQUEST_RETRY_NUMBER:
     ERROR_LOG(DVDINTERFACE, "DVDLowRequestRetryNumber");
+    DolphinAnalytics::Instance().ReportGameQuirk(GameQuirk::USES_DVD_LOW_REQUEST_RETRY_NUMBER);
+    SetHighError(ERROR_INV_CMD);
+    interrupt_type = INT_DEINT;
     break;
   // Probably only used by Wii
-  case DVDLowSetMaximumRotation:
+  case DICommand::SET_MAXIMUM_ROTATION:
     ERROR_LOG(DVDINTERFACE, "DVDLowSetMaximumRotation");
+    SetHighError(ERROR_INV_CMD);
+    interrupt_type = INT_DEINT;
     break;
   // Probably only used by Wii
-  case DVDLowSerMeasControl:
+  case DICommand::SER_MEAS_CONTROL:
     ERROR_LOG(DVDINTERFACE, "DVDLowSerMeasControl");
+    DolphinAnalytics::Instance().ReportGameQuirk(GameQuirk::USES_DVD_LOW_SER_MEAS_CONTROL);
+    SetHighError(ERROR_INV_CMD);
+    interrupt_type = INT_DEINT;
     break;
 
   // Used by both GC and Wii
-  case DVDLowRequestError:
+  case DICommand::REQUEST_ERROR:
     INFO_LOG(DVDINTERFACE, "Requesting error... (0x%08x)", s_error_code);
     WriteImmediate(s_error_code, output_address, reply_to_ios);
-    s_error_code = 0;
+    SetHighError(0);
     break;
 
   // Audio Stream (Immediate). Only seems to be used by some GC games
   // (command_0 >> 16) & 0xFF = Subcommand
   // command_1 << 2           = Offset on disc
   // command_2                = Length of the stream
-  case 0xE1:
+  case DICommand::PLAY_AUDIO_STREAM:
   {
-    u8 cancel_stream = (command_0 >> 16) & 0xFF;
-    if (cancel_stream)
+    INFO_LOG(DVDINTERFACE, "(Audio) Stream cmd: %08x offset: %08" PRIx64 " length: %08x", command_0,
+             static_cast<u64>(command_1) << 2, command_2);
+
+    if (!s_enable_dtk)
     {
-      s_stop_at_track_end = false;
-      s_stream = false;
+      ERROR_LOG(DVDINTERFACE, "Attempted to play audio while audio is disabled!");
+      SetHighError(ERROR_AUDIO_BUF);
+      interrupt_type = INT_DEINT;
+      break;
     }
-    else
+
+    switch ((command_0 >> 16) & 0xFF)
     {
+    case 0x00:
```

Comment:
Do we want to _name_ this case (as well as `0x01` further down)? Or at least leave a comment.
All the other cases either have an `enum` (or `constexpr`) name, a comment next to them; or just an `ERROR_LOG` that makes it immediately obvious what it means.
Main reason is that Dolphin might be the only place out there that _knows_ what the values mean, and serve as documentation for others.

Reply:
I'll split the earlier log message into two messages for the cases.  It should be split anyways because the parameters are ignored in case 0x01 (stopping the stream).